cat-theme/src/templates/header/hero.ftl

```diff
@@ -0,0 +1,31 @@
+<header class="hero parallax" id="hero">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-2 d-md-block d-none hero-carousal">
+            </div>
```

Comment:
The closing tag can be on the same line as the opening tag if there is no content inside.